### PR TITLE
Adding Turbinia scaffolder for generating jobs and tasks.

### PR DESCRIPTION
**Pull request by kiddinn**
Adding a new scaffolder to generate jobs and tasks for Turbinia, part of #49

### CODE REVIEW

**Comment by kiddinn** [comment]
new request for the scaffolder, @aarontp can you take a look

File: l2tscaffolder/scaffolders/turbinia.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  PROJECT = definitions.DEFINITION_TURBINIA
+
+  # Filename of templates.
+  TEMPLATE_JOB_FILE = None
+  TEMPLATE_TASK_FILE = None
+
+  def __init__(self):
+    """Initializes the Turbinia scaffolder."""
+    super(TurbiniaBaseScaffolder, self).__init__()
+    self._plugin_path = os.path.join('turbinia', 'jobs')
```
Comment (Onager) [inline l2tscaffolder/scaffolders/turbinia.py]: Please change all the references from "plugin" to "job" to make it clear what's actually being generated here.

File: l2tscaffolder/scaffolders/turbinia.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    self._task_path = os.path.join('turbinia', 'workers')
+    self._mapping_helper = mapping_helper.MappingHelper()
+
+    self.class_name = ''
+
+  def _GenerateJobFile(self) -> str:
```
Comment (Onager) [inline l2tscaffolder/scaffolders/turbinia.py]: Shouldn't these methods generating job and tasks being in job and task specific scaffolders, not the shared parent class?

File: l2tscaffolder/templates/turbinia_job.jinja2 [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  # The types of evidence that this Job will process
+  # TODO: Fill this out.
+  evidence_input = []
+  evidence_output = []
+
+  NAME = '{{ class_name }}Job'
+
+  def create_tasks(self, evidence):
+    """Create task.
```
Comment (Onager) [inline l2tscaffolder/templates/turbinia_job.jinja2]: "Create tasks"

**Pull request merged**